### ISSUE TITLE
qa: Intermittent failure in interface_zmq.py "Resource temporarily unavailable"

### ISSUE THREAD

**Issue by hebasto** [bitcoin/bitcoin#20590]
https://cirrus-ci.com/task/4685360343023616?command=ci#L1078
```
50/194 - interface_zmq.py failed, Duration: 97 s
stdout:
2020-12-07T13:33:26.675000Z TestFramework (INFO): Initializing test directory /tmp/cirrus-ci-build/ci/scratch/test_runner/test_runner_₿_🏃_20201207_132422/interface_zmq_150
2020-12-07T13:33:50.656000Z TestFramework (INFO): Generate 5 blocks (and 5 coinbase txes)
2020-12-07T13:33:51.912000Z TestFramework (INFO): Wait for tx from second node
2020-12-07T13:33:55.422000Z TestFramework (INFO): Test the getzmqnotifications RPC
2020-12-07T13:33:55.428000Z TestFramework (INFO): Testing 'sequence' publisher
2020-12-07T13:34:02.510000Z TestFramework (INFO): Wait for tx from second node
2020-12-07T13:34:04.838000Z TestFramework (INFO): Testing sequence notifications with mempool sequence values
2020-12-07T13:34:04.838000Z TestFramework (INFO): Testing RBF notification
2020-12-07T13:34:19.694000Z TestFramework (INFO): Testing reorg notifications
2020-12-07T13:34:21.807000Z TestFramework (INFO): Evict mempool transaction by block conflict
2020-12-07T13:34:27.378000Z TestFramework (INFO): Testing 'mempool sync' usage of sequence notifier
2020-12-07T13:35:02.427000Z TestFramework (ERROR): Unexpected exception caught during testing
Traceback (most recent call last):
  File "/tmp/cirrus-ci-build/ci/scratch/build/bitcoin-x86_64-pc-linux-gnu/test/functional/test_framework/test_framework.py", line 126, in main
    self.run_test()
  File "/tmp/cirrus-ci-build/ci/scratch/build/bitcoin-x86_64-pc-linux-gnu/test/functional/interface_zmq.py", line 76, in run_test
    self.test_reorg()
  File "/tmp/cirrus-ci-build/ci/scratch/build/bitcoin-x86_64-pc-linux-gnu/test/functional/interface_zmq.py", line 201, in test_reorg
    assert_equal(self.nodes[0].getbestblockhash(), hashblock.receive().hex())
  File "/tmp/cirrus-ci-build/ci/scratch/build/bitcoin-x86_64-pc-linux-gnu/test/functional/interface_zmq.py", line 37, in receive
    topic, body, seq = self.socket.recv_multipart()
  File "/usr/lib/python3/dist-packages/zmq/sugar/socket.py", line 475, in recv_multipart
    parts = [self.recv(flags, copy=copy, track=track)]
  File "zmq/backend/cython/socket.pyx", line 791, in zmq.backend.cython.socket.Socket.recv
  File "zmq/backend/cython/socket.pyx", line 827, in zmq.backend.cython.socket.Socket.recv
  File "zmq/backend/cython/socket.pyx", line 191, in zmq.backend.cython.socket._recv_copy
  File "zmq/backend/cython/socket.pyx", line 186, in zmq.backend.cython.socket._recv_copy
  File "zmq/backend/cython/checkrc.pxd", line 19, in zmq.backend.cython.checkrc._check_rc
zmq.error.Again: Resource temporarily unavailable
2020-12-07T13:35:02.479000Z TestFramework (INFO): Stopping nodes
2020-12-07T13:35:03.401000Z TestFramework (WARNING): Not cleaning up dir /tmp/cirrus-ci-build/ci/scratch/test_runner/test_runner_₿_🏃_20201207_132422/interface_zmq_150
2020-12-07T13:35:03.402000Z TestFramework (ERROR): Test failed. Test logging available at /tmp/cirrus-ci-build/ci/scratch/test_runner/test_runner_₿_🏃_20201207_132422/interface_zmq_150/test_framework.log
2020-12-07T13:35:03.402000Z TestFramework (ERROR): 
2020-12-07T13:35:03.402000Z TestFramework (ERROR): Hint: Call /tmp/cirrus-ci-build/ci/scratch/build/bitcoin-x86_64-pc-linux-gnu/test/functional/combine_logs.py '/tmp/cirrus-ci-build/ci/scratch/test_runner/test_runner_₿_🏃_20201207_132422/interface_zmq_150' to consolidate all logs
2020-12-07T13:35:03.402000Z TestFramework (ERROR): 
2020-12-07T13:35:03.403000Z TestFramework (ERROR): If this failure happened unexpectedly or intermittently, please file a bug and provide a link or upload of the combined log.
2020-12-07T13:35:03.403000Z TestFramework (ERROR): https://github.com/bitcoin/bitcoin/issues
2020-12-07T13:35:03.403000Z TestFramework (ERROR): 
```

**Comment by MarcoFalke** [comment]
Discussion in #20538

**Comment by adamjonas** [comment]
I think this can be closed so that future comments are pointed to #20538 and #20934 to focus the discussion.
